src/tests/api/HappyPath.test.ts

```diff
@@ -0,0 +1,244 @@
+import { sessionPost, stubStartPost, postDocumentSelection, startStubServiceAndReturnSessionId, getSessionById, getSessionAndVerifyKey, authorizationGet, tokenPost, userInfoPost, sessionConfigurationGet, postAbortSession } from "./ApiTestSteps";
+import { getTxmaEventsFromTestHarness, validateTxMAEventData } from "./ApiUtils";
+import f2fStubPayload from "../data/exampleStubPayload.json";
+import thinFilePayload from "../data/thinFilePayload.json";
+import exampleStubPayload from "../data/exampleStubPayload.json";
+import abortPayload from "../data/abortPayload.json";
+import dataPassport from "../data/docSelectionPayloadPassportValid.json";
+import dataUkDrivingLicence from "../data/docSelectionPayloadDriversLicenceValid.json";
+import dataEuDrivingLicence from "../data/docSelectionPayloadEuDriversLicenceValid.json";
+import dataNonUkPassport from "../data/docSelectionPayloadNonUkPassportValid.json";
+import dataBrp from "../data/docSelectionPayloadBrpValid.json";
+import dataEeaIdCard from "../data/docSelectionPayloadEeaIdCardValid.json";
+import { constants } from "./ApiConstants";
+import { DocSelectionData } from "./types";
+
+describe("/session endpoint", () => {
+
+	it("Successful Request Tests", async () => {
+		const stubResponse = await stubStartPost(exampleStubPayload);
+		const postRequest = await sessionPost(stubResponse.data.clientId, stubResponse.data.request);
+		expect(postRequest.status).toBe(200);
+		const sessionId = postRequest.data.session_id;
+
+		await getSessionAndVerifyKey(sessionId, constants.DEV_F2F_SESSION_TABLE_NAME, "authSessionState", "F2F_SESSION_CREATED");
+
+		const allTxmaEventBodies = await getTxmaEventsFromTestHarness(sessionId, 1);
+		validateTxMAEventData({ eventName: "F2F_CRI_START", schemaName: "F2F_CRI_START_SCHEMA" }, allTxmaEventBodies);
+	});
+});
+
+describe("/documentSelection Endpoint", () => {
+	it.each([
+		{ yotiMockId: "0000", docSelectionData: dataUkDrivingLicence, yotiStartSchema: "F2F_YOTI_START_00_SCHEMA" },
+		{ yotiMockId: "0100", docSelectionData: dataPassport, yotiStartSchema: "F2F_YOTI_START_SCHEMA" },
+		{ yotiMockId: "0200", docSelectionData: dataNonUkPassport, yotiStartSchema: "F2F_YOTI_START_SCHEMA" },
+		{ yotiMockId: "0300", docSelectionData: dataBrp, yotiStartSchema: "F2F_YOTI_START_03_SCHEMA" },
+		{ yotiMockId: "0400", docSelectionData: dataEuDrivingLicence, yotiStartSchema: "F2F_YOTI_START_00_SCHEMA" },
+		{ yotiMockId: "0500", docSelectionData: dataEeaIdCard, yotiStartSchema: "F2F_YOTI_START_05_SCHEMA" },
+	])("Successful Request Tests - $yotiMockId", async ({ yotiMockId, docSelectionData, yotiStartSchema }: { yotiMockId: string; docSelectionData: DocSelectionData; yotiStartSchema: string }) => {
+		f2fStubPayload.yotiMockID = yotiMockId;
```

Comment:
Shall we use `structuredClone` here so that we're not directly mutating an object that is being reused like we do on  line 79?

Reply:
Updated test logic so we're using a structuresClone in all tests where values are being updated in the f2fStubPayload object